### PR TITLE
github: add template for dm and refine other templates

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 

close https://github.com/pingcap/ticdc/issues/3175

### What is changed and how it works?

add template for dm and refine other templates.

### Check List 

Tests 

 - No code

Code changes

None

Side effects

None

Related changes
None

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by hi-rustin** [comment]
/hold

Waiting for https://github.com/ti-community-infra/configs/pull/456 landed.

**Comment by hi-rustin** [comment]
/unhold

Ready for review!

File: .github/ISSUE_TEMPLATE/question.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ body:
           required: true
         - label: Searched open and closed [GitHub issues](https://github.com/pingcap/ticdc/issues?q=is%3Aissue)
           required: true
-        - label:  Read the [documentation](https://docs.pingcap.com/tidb/stable/ticdc-overview)
+        - label:  Read the [documentation](https://docs.pingcap.com/tidb/stable)
```
Comment (lance6716) [inline .github/ISSUE_TEMPLATE/question.yml:16]: not sure why we changed the URL

File: .github/ISSUE_TEMPLATE/dm-bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+             ```console
+             (paste upstream MySQL/MariaDB server version here)
+             ```
+             Downstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):
+             ```console
+             (paste TiDB cluster version here)
+             ```
+             How did you deploy DM: tiup or manually?
+             ```console
+             (leave DM-Ansible or manually here)
```
Comment (lance6716) [inline .github/ISSUE_TEMPLATE/dm-bug-report.yml:47]:
```suggestion
             (leave TiUP or manually here)
```

File: .github/ISSUE_TEMPLATE/dm-bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+             ```console
+             >
+             >
+             ```
+    validations:
+      required: true
+  - type: textarea
+    id: current
+    attributes:
+      label: current status of DM cluster (execute `query-status` in dmctl)
```
Comment (lance6716) [inline .github/ISSUE_TEMPLATE/dm-bug-report.yml:59]:
```suggestion
      label: current status of DM cluster (execute `query-status <task-name>` in dmctl)
```

**Review by lance6716 — COMMENTED** [review]
rest lgtm

File: .github/ISSUE_TEMPLATE/question.yml (hunk 1 above)
Comment (hi-rustin) [reply .github/ISSUE_TEMPLATE/question.yml:16]: Since this is a shared template, it may not be appropriate to jump directly to ticdc's documentation.

**Review by lance6716 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/dm-bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+          - Other interesting logs
+          - Output of dmctl's commands with problems
+  - type: markdown
+    attributes:
+      value: |
+        Configuration of the cluster and the task
+          - `dm-worker.toml` for every DM-worker instance if possible
+          - `dm-master.toml` for DM-master if possible
+          - task config, like `task.yaml` if possible
+          - `inventory.ini` if deployed by DM-Ansible
```
Comment (amyangfei) [inline .github/ISSUE_TEMPLATE/dm-bug-report.yml:81]: Do we still need this line @lance6716

Comment (lance6716) [reply .github/ISSUE_TEMPLATE/dm-bug-report.yml:81]: no 👍

**Review by lichunzhu — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/dm-bug-report.yml (hunk 4 above)
Comment (hi-rustin) [inline .github/ISSUE_TEMPLATE/dm-bug-report.yml:81]:
```suggestion
```

Comment (hi-rustin) [reply .github/ISSUE_TEMPLATE/dm-bug-report.yml]: Done!

**Review by amyangfei — APPROVED** [review]

**Pull request merged**